### PR TITLE
K8SPG-352 add pgo 2.0.1 for dbaas

### PR DESCRIPTION
**Pull request by nmarukovich**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: api-tests/apply_route_test.go [hunk 1]
```diff
@@ -546,12 +546,20 @@ func TestApplyPGReturnedVersions(t *testing.T) {
...
+		{"latest", "2.0.1", "13.10", "13.10"},
+		{"latest", "2.0.1", "12.14", "12.14"},
 		{"latest", "2.0.0", "", "14.6"},
 		{"latest", "2.0.0", "12.3", "12.13"},
 		{"latest", "2.0.0", "13.9", "13.9"},
 		{"latest", "2.0.0", "14.6", "14.6"},
 
 		// test recommended
+		{"recommended", "2.0.1", "", "15.2"},
+        {"recommended", "2.0.1", "14.7", "14.7"},
```
Comment (tplavcic) [inline api-tests/apply_route_test.go:560]: indentation issue?

File: api-tests/apply_route_test.go [hunk 2]
```diff
@@ -567,6 +575,10 @@ func TestApplyPGReturnedVersions(t *testing.T) {
 		{"12.14", "1.4.0", "", "12.14"},
 		{"13.10", "1.4.0", "", "13.10"},
 		{"14.7", "1.4.0", "", "14.7"},
+		{"15.2", "2.0.1", "", "15.2"},
+        {"14.7", "2.0.1", "", "14.7"},
```
Comment (tplavcic) [inline api-tests/apply_route_test.go:579]: indentation issue?

File: api-tests/apply_route_test.go [hunk 3]
```diff
@@ -584,8 +596,16 @@ func TestApplyPGReturnedVersions(t *testing.T) {
...
 		{"13-latest", "2.0.0", "", "13.9"},
 		{"14-latest", "2.0.0", "", "14.6"},
+		{"12-latest", "2.0.1", "", "12.14"},
+		{"13-latest", "2.0.1", "", "13.10"},
+		{"14-latest", "2.0.1", "", "14.7"},
+		{"15-latest", "2.0.1", "", "15.2"},
 
 		// test with distribution suffix
+		{"latest", "2.0.1", "12.14 - Percona Distribution", "12.14"},
+        {"latest", "2.0.1", "13.10 - Percona Distribution", "13.10"},
```
Comment (tplavcic) [inline api-tests/apply_route_test.go:606]: indentation issue?

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by egegunes — APPROVED** [review]

**Pull request merged**